deploy/deploy.yml

```diff
@@ -0,0 +1,85 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: event-exporter
+  labels:
+    name: event-exporter
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    name: event-exporter
+  name: event-exporter
+subjects:
+  - kind: ServiceAccount
+    name: event-exporter
+    namespace: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: view
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    name: event-exporter
+  name: event-exporter
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: event-exporter
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        prometheus.io/path: /metrics
+        prometheus.io/port: '9102'
+        prometheus.io/scrape: 'true'
+      labels:
+        app: event-exporter
+    spec:
+      containers:
+        - name: event-exporter
+          image: 'caicloud/event-exporter:v1.0.0'
+          imagePullPolicy: Always
+          args:
+            - --eventType=Warning
+            - --eventType=Normal
+          ports:
+            - containerPort: 9102
+              name: http
+          resources:
+            limits:
+              memory: 100Mi
+            requests:
+              memory: 40Mi
+      serviceAccountName: event-exporter
+      terminationGracePeriodSeconds: 30
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/scrape: 'true'
+    service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
+    service.beta.kubernetes.io/aws-load-balancer-ssl-ports: https
+  labels:
+    name: event-exporter
+  name: event-exporter
+spec:
+  type: NodePort
```

Comment:
not a recommended way to expose service port

Reply:
fixed